lib/config/guiconfiguration.go

```diff
@@ -13,17 +13,27 @@ import (
 )
 
 type GUIConfiguration struct {
-	Enabled                   bool   `xml:"enabled,attr" json:"enabled" default:"true"`
-	RawAddress                string `xml:"address" json:"address" default:"127.0.0.1:8384"`
-	User                      string `xml:"user,omitempty" json:"user"`
-	Password                  string `xml:"password,omitempty" json:"password"`
-	RawUseTLS                 bool   `xml:"tls,attr" json:"useTLS"`
-	APIKey                    string `xml:"apikey,omitempty" json:"apiKey"`
-	InsecureAdminAccess       bool   `xml:"insecureAdminAccess,omitempty" json:"insecureAdminAccess"`
-	Theme                     string `xml:"theme" json:"theme" default:"default"`
-	Debugging                 bool   `xml:"debugging,attr" json:"debugging"`
-	InsecureSkipHostCheck     bool   `xml:"insecureSkipHostcheck,omitempty" json:"insecureSkipHostcheck"`
-	InsecureAllowFrameLoading bool   `xml:"insecureAllowFrameLoading,omitempty" json:"insecureAllowFrameLoading"`
+	Enabled                   bool              `xml:"enabled,attr" json:"enabled" default:"true"`
+	RawAddress                string            `xml:"address" json:"address" default:"127.0.0.1:8384"`
+	User                      string            `xml:"user,omitempty" json:"user"`
+	Password                  string            `xml:"password,omitempty" json:"password"`
+	AuthMode                  AuthMode          `xml:"authMode,omitempty" json:"authMode"`
+	LDAPAuth                  LDAPConfiguration `xml:"ldapAuth,omitempty" json:"ldapAuth"`
```

Comment:
I would move LDAPConfigurarion as LDAPConfiguration to Config instead of GUIConfiguration.